arangod/Cluster/Maintenance.cpp

```diff
@@ -462,10 +495,10 @@ arangodb::Result arangodb::maintenance::diffPlanLocal(
   for (auto const& ldb : VPackObjectIterator(local)) {
     auto const& dbname = ldb.key.copyString();
     if (!plan.hasKey(std::vector<std::string>{DATABASES, dbname})) {
-      actions.emplace_back(
-          ActionDescription({{std::string(NAME), std::string(DROP_DATABASE)},
-                             {std::string(DATABASE), std::string(dbname)}},
-                            HIGHER_PRIORITY));
+      actions.emplace_back(ActionDescription(
+          std::map<std::string, std::string>{{std::string(NAME), std::string(DROP_DATABASE)}, {std::string("tick"), std::to_string(TRI_NewTickServer())},
```

Comment:
The "tick" value here will always be unique and will not be equal to the database's id. What's its purpose?
I think it deserves a code comment.

Reply:
tick attribute removed